cdhweb/pages/management/commands/exodus.py

```diff
@@ -353,5 +430,17 @@ def get_media_files(self):
     def get_wagtail_image(self, image):
         # get the migrated wagtail image for a foreign-key image
         # using image file basename, which is migrated as image title
-        return Image.objects.get(title=os.path.basename(image.name))
-
+        try:
+            return Image.objects.get(title=os.path.basename(image.name))
+        except Image.DoesNotExist as err:
+            print("Error locating image %s: %s" % (image, err))
+            return None
```

Comment:
Do we need to do some error handling related to this, and is that work tracked anywhere? (Are these some of the images that aren't migrating properly?)

Reply:
I'm not sure...I don't understand the reasons why images would be missing well enough; I'd defer to you (but yes that's what these are). Overall it would be good to have more real logging/handling for the exodus script (maybe we could improve the messaging about the corrupt EXIF data too).

Reply:
ok, good point — probably the real place to look is the image exodus portion. Should that be handled during tested or should we add a chore to investigate the handful of images that are not being converted?

Reply:
probably chore, I don't think it should block release of the image exodus feature.

Reply:
That makes sense to me (as long as we make it part of 3.0); and we may figure more things out once we start doing acceptance testing and getting another set of eyes on all of this.

Reply:
created #271